docs/gt4py/quickstart.rst

```diff
@@ -183,29 +185,43 @@ set of routines used in `NumPy` for allocation: ``from_array``, ``ones``, ``zero
         data=np.random.randn(10, 10, 10),
         backend=backend,
         dtype=np.float64,
-        default_origin=(0, 0, 0),
+        aligned_index=(0, 0, 0),
     )
     field_b = gt_storage.ones(
-        backend=backend, shape=(10, 10, 10), dtype=np.float64, default_origin=(0, 0, 0)
+        backend=backend, shape=(10, 10, 10), dtype=np.float64, aligned_index=(0, 0, 0)
     )
     field_c = gt_storage.zeros(
-        backend=backend, shape=(10, 10, 10), dtype=np.float64, default_origin=(0, 0, 0)
+        backend=backend, shape=(10, 10, 10), dtype=np.float64, aligned_index=(0, 0, 0)
     )
     result = gt_storage.empty(
-        backend=backend, shape=(10, 10, 10), dtype=np.float64, default_origin=(0, 0, 0)
+        backend=backend, shape=(10, 10, 10), dtype=np.float64, aligned_index=(0, 0, 0)
     )
 
     stencil_example(field_a, field_b, field_c, result, alpha=0.5)
 
 
-The ``default_origin`` parameter plays two roles:
+The ``aligned_index`` specifies that the array is to be allocated such that memory address of the point specified in
+``aligned_index`` is `aligned` to a backend-dependent value. For optimal performance, you set the ``algined_index`` to
+a point which is the lower-left corner of the iteration domain most frequently used for this field.
 
-#. The data is allocated such that memory address of the point specified in ``default_origin`` is `aligned` to a
-   backend-dependent value. For optimal performance, you set the ``default_origin`` to a point which is the
-   lower-left corner of the iteration domain most frequently used for this storage.
+----------------
+Array Interfaces
+----------------
 
-#. If when calling the stencil, no other `origin` is specified, this value is where the `iteration domain` begins, i.e.
-   the grid point with the lowest index where a value is written.
+When passing buffers to stencils, they can be in any form that is compatible with ``np.asarray`` or ``cp.asarray``,
```

Comment:
Is the idea that it's really anything that implements the array interface protocol (buffer protocol) https://numpy.org/doc/stable/reference/arrays.interface.html?

Reply:
In principle, yes. e.g. `numpy` backends and `dace:*` backends should be fine with that. The option to reject a buffer based on wrong layout, (including no contiguous dimension) is still there!

Reply:
Great!